src/menus/LabelMenus.cpp

```diff
@@ -237,9 +249,25 @@ void EditClipboardByLabel(AudacityProject &project,
    if (regions.empty())
       return;
 
-   const bool notLocked = (!SyncLockState::Get(project).IsSyncLocked() &&
-                           (tracks.Selected<PlayableTrack>()).empty());
+   const bool syncLocked = SyncLockState::Get(project).IsSyncLocked();
+   const bool playableTrackSelected = !tracks.Selected<PlayableTrack>().empty();
 
+   const auto tracksToEdit = tracks.Any<Track>() + [&](const auto pTrack) {
+      // All sync-locked tracks should be edited,
```

Comment:
Predicate could be extracted into predicate builder function to avoid code duplication in the same file

Reply:
makes sense, done